.github/workflows/main.yml

```diff
@@ -14,6 +14,21 @@ defaults:
     shell: bash -el {0}
 
 jobs:
+
+  shellcheck:
```

Comment:
Is this worth having? I think we only have the run-local.sh file

Reply:
It does not hurt